.github/CODEOWNERS

```diff
@@ -331,7 +331,7 @@
 # Kibana Platform Security
 /packages/kbn-crypto/ @elastic/kibana-security
 /packages/kbn-handlebars/ @elastic/kibana-security
-/src/core/server/http/csp/ @elastic/kibana-security @elastic/kibana-core
+/packages/core/http/core-http-server-internal/src/csp/ @elastic/kibana-security @elastic/kibana-core
```

Comment:
Just out of curiosity, what happens when there are multiple paths that match the same file?
I'm asking cause I've seen that `/packages/core/` is also defined (in line 286 ATM), and I wonder if it's a first match, last match, override, merge....

Reply:
It's a last match IIRC